stock_picking_batch_extended/migrations/12.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from openupgradelib import openupgrade
+
+
+_field_renames = [
+    'stock.warehouse', 'stock_warehouse', 'default_picker_id',
```

Comment:
Where is defined stock.wharehouse?

Reply:
I don't get you... `stock` module defines `stock.warehouse`, but this is only renaming a field..

Reply:
Upps!! ok... I see.. model, table, old_field, renamed_field...
Sorry